test/junit/scala/collection/SetMapConsistencyTest.scala

```diff
@@ -506,7 +506,7 @@ class SetMapConsistencyTest {
       else None
     }
     assert(f() match {
-      case Some((a,b)) if (a==null || b==null) => false
+      case Some((null, _)) => false
```

Comment:
`case Some((a,b)) => a != null && b != null`? w/o reading the test.

Reply:
the type of `b` is `Int`, so the null check isn't necessary